spyder/plugins/pylint/main_widget.py

```diff
@@ -315,6 +316,14 @@ def __init__(self, name=None, plugin=None, parent=None):
         self.datelabel.ID = PylintWidgetToolbarItems.DateLabel
 
         self.treewidget = ResultsTree(self)
+        self.pane_empty = PaneEmptyWidget(
+            self,
+            "code-analysis",
+            _("No code analyzed yet"),
```

Comment:
```suggestion
            _("Code not analyzed yet"),
```

Likewise, it doesn't make sense to say that no code was analyzed yet, if this appears whenever the user clicks any individual _file_ that has not yet been analyzed, whether or not other code was analyzed previously. However, a few tweaks gives a much clearer and more appropriate message (which can also be adopted for the Profiler pane if/when that adopts similar behavior).